.github/workflows/build.yaml

```diff
@@ -15,7 +15,11 @@ jobs:
         uses: Randomblock1/theos-action@v1
 
       - name: Build package
-        run: make package
+        run: |
+          export PACKAGE_BUILDNAME=$(git rev-parse --short HEAD)-rootless
+          make package THEOS_PACKAGE_SCHEME=rootless FINALPACKAGE=1
```

Comment:
For some reason that we have yet to understand, this package doesn't seem to work on any of our test devices when we build with `FINALPACKAGE=1`. Have you tested that this works on one of your test devices with that option enabled?

Reply:
FYI when toggling between compiling for rootful and rootless, you need to `make clean` instead of just making the package. For instance, this is how I compile for both rootful and rootless in my jailbreak tweaks:

For rootful:
```bash
make clean package FINALPACKAGE=1 
```

For rootless:
```bash
make clean package THEOS_PACKAGE_SCHEME=rootless FINALPACKAGE=1
```

This may or not be the issue. I'm putting this out there since it stumped me for a while too. If you do not `make clean` in between switching the target scheme, the tweak will not function on device. Hope this helps.

Reply:
Were you able to successfully run this tweak in on a rootful device? I cannot get it to work and I don't have a rootless option for my older device.